app/auth/apikey_crud.py

```diff
@@ -67,13 +67,14 @@ def __init__(self) -> None:
             Column("api_key", String, primary_key=True, index=True),
             Column("name", String),
             Column("user_id", String, index=True, nullable=False),
+            Column("user_login", String),
```

Comment:
In RSPY, we need the user login (e.g. "jgaucher") associated with the api key because it is used as the default owner_id to create collections in the catalog. The user_id (e.g. "6643c0f6-06e0-4ed2-9dbb-6af947582730") is not user-friendly.